sw/device/lib/dif/dif_aes.h

```diff
@@ -147,12 +196,12 @@ typedef enum dif_aes_manual_operation {
    * AES operates in automatic mode - which means that the encryption/decryption
    * is automatically triggered on every successful `dif_aes_*_load_data()`.
    */
-  kDifAesManualOperationAuto = 0,
+  kDifAesManualOperationAuto = 1,
```

Comment:
This needs to be reverted. the MANUAL_OPERATION field/bit is not one-hot encoded. 0 means automatic operation, 1 means manual operation.

Reply:
Done.